test/libsolidity/syntaxTests/array/invalid/library_index_access.sol

```diff
@@ -4,4 +4,4 @@ library C {
     }
 }
 // ----
-// TypeError 2876: (51-55): Index access for contracts or libraries is not possible.
+// TypeError 2876: (51-55): Index access for library types and arrays of libraries are not possible.
```

Comment:
Arrays of libraries?

Reply:
What would you call the type of ``L[4] memory x;``? That's disallowed by the other one by now, but we still have ``abi.decode(x,(L[4]));`` that's not an index access, but an array type.

Reply:
Yea, was just confused but the other comment clarified it